### ISSUE TITLE
Implement shortnames

### ISSUE THREAD

**Issue by ericcurtin** [containers/ramalama#38]
podman has this concept of shortnames, which looks like this:

/etc/containers/registries.conf.d/000-shortnames.conf

```
$ head /etc/containers/registries.conf.d/000-shortnames.conf
[aliases]
  # almalinux
  "almalinux" = "docker.io/library/almalinux"
  "almalinux-minimal" = "docker.io/library/almalinux-minimal"
  # Amazon Linux
  "amazonlinux" = "public.ecr.aws/amazonlinux/amazonlinux"
  # Arch Linux
  "archlinux" = "docker.io/library/archlinux"
  # centos
  "centos" = "quay.io/centos/centos"
  ```

Create something similar `/etc/ramalama/registries.conf.d/000-shortnames.conf` that can do the same for ramalama

Suggested by @rhatdan in https://github.com/containers/ramalama/pull/35

**Comment by rhatdan** [comment]
I think this is a good idea.

**Comment by ericcurtin** [comment]
The podman file for this is:

$ grep -v "^#" /etc/containers/registries.conf
unqualified-search-registries = ["registry.fedoraproject.org", "registry.access.redhat.com", "docker.io"]
short-name-mode = "enforcing"

so I guess the ramalama equivalent would be:

/etc/ramalama/registries.conf

**Comment by rhatdan** [comment]
Yes I think that makes sense, although perhaps we should use /usr/share/ramalama/registries.conf and then allow sysadmins to override in /etc/ramalama.
